docs/getting_started/out_of_core.rst

```diff
@@ -1,51 +1,58 @@
-Out of Core in Modin
-====================
+Out-of-memory data with Modin
+=============================
 
-If you are working with very large files or would like to exceed your memory, you may
-change the primary location of the `DataFrame`_. If you would like to exceed memory, you
-can use your disk as an overflow for the memory.
+When using pandas, you might run into a memory error if you are working with large datasets that cannot fit in memory or perform certain memory-intensive operations (e.g., joins). 
 
-Starting Modin with out of core enabled
----------------------------------------
+Modin solves this problem by spilling over to disk, in other words, it uses your disk as an overflow for memory so that you can work with datasets that are too large to fit in memory. By default, Modin leverages out-of-core methods to handle datasets that don't fit in memory for both Ray and Dask engines.
 
-Out of core is now enabled by default for both Ray and Dask engines.
 
-Disabling Out of Core
----------------------
+Motivating Example: Memory error with pandas
+--------------------------------------------
 
-Out of core is enabled by the compute engine selected. To disable it, start your
-preferred compute engine with the appropriate arguments. For example:
+pandas makes use of in-memory data structures to store and operate on data, which means that if you have a dataset that is too large to fit in memory, it will cause an error on pandas. As an example, let's creates a 80GB DataFrame by appending together 40 different 2GB DataFrames. 
 
 .. code-block:: python
 
-  import modin.pandas as pd
-  import ray
+  import pandas as old_pd
+  import numpy as np
+  df = old_pd.concat([old_pd.DataFrame(np.random.randint(0, 100, size=(2**20, 2**8))) for _ in range(40)]) # Memory Error!
```

Comment:
```suggestion
  import pandas as pd
  import numpy as np
  df = pd.concat([old_pd.DataFrame(np.random.randint(0, 100, size=(2**20, 2**8))) for _ in range(40)]) # Memory Error!
```
It is unclear to me what `old_pd` stands for so I would prefer the use of `pd`.

Reply:
After this is fixed, LGTM.

Reply:
I prefer it stay with `old_pd` to differentiate between `modin.pandas as pd` and `pandas as old_pd` in places where we are comparing against pandas in the documentation.

Reply:
We say everywhere: hey, a user, you just need to replace your import statement from `import pandas as pd` to `import modin.pandas as pd` to scale pandas queries. There is no `old_pd` reference anywhere. That might be confusing for users (for me, too). There might be a question - What does `old_pd` mean? Is that a old pandas version?

Reply:
What if instead of `old_pd` we say, this is what happens if you use pandas: 
```python
import pandas
import numpy as np
df = pandas.concat([old_pd.DataFrame(np.random.randint(0, 100, size=(2**20, 2**8))) for _ in range(40)]) # Memory Error!
```
That way there is no namespace conflict if the user copies and paste both this code and the modin code into the same notebook context to try it out.

Reply:
I am okay with this.

Reply:
@devin-petersohn, thoughts?

Reply:
Ok with me too.

Reply:
Fixed!